modules/50_damages/KWLike/realization.gms

```diff
@@ -1,13 +1,7 @@
 
 *####################### R SECTION START (PHASES) ##############################
-$Ifi "%phase%" == "sets" $include "./modules/50_damages/KWLike/sets.gms"
```

Comment:
These changes seem somehow unrelated.

Reply:
Yes, it is the automated code-update which were not pushed together with the code-changes